website/content/docs/k8s/helm.mdx

```diff
@@ -211,25 +212,25 @@ Use these links to navigate to a particular top-level stanza.
 
         - `secretKey` ((#v-global-secretsbackend-vault-ca-secretkey)) (`string: ""`) - The key within the Kubernetes or Vault secret that holds the Vault CA certificate.
 
-      - `connectCA` ((#v-global-secretsbackend-vault-connectca)) - Configuration for the Vault service mesh CA provider.
+      - `connectCA` ((#v-global-secretsbackend-vault-connectca)) - Configuration for the Vault Connect CA provider.
```

Comment:
@jkirschner-hashicorp and I updated "Consul Connect" -> "service mesh" which a more appropriate name for this. is there a reason why we're reverting it back?